### PR TITLE
Improve the info command to show information about the logic trees

### PR DESCRIPTION
**Pull request by micheles**
This is useful for debugging. I am also adding documentation for the event based calculator.
The tests are green: https://ci.openquake.org/job/zdevel_oq-risklib/378

### CODE REVIEW

File: openquake/commonlib/source.py [hunk 1]
```diff
@@ -83,6 +95,25 @@ class TrtModel(collections.Sequence):
...
+        """
+        source_stats_dict = {}
+        for src in sources:
+            trt = src['tectonicRegion']
+            if trt not in source_stats_dict:
+                source_stats_dict[trt] = TrtModel(trt)
+            tm = source_stats_dict[trt]
+            if not tm.sources:
+                tm.num_ruptures = 1
+                tm.sources.append(src)
```
Comment (ptormene) [inline openquake/commonlib/source.py]: Please check this

File: openquake/commonlib/source.py [hunk 2]
```diff
@@ -432,6 +478,7 @@ def __init__(self, source_model_lt, source_models):
         if len(list(self.sources)) == 0:
             raise RuntimeError('All sources were filtered away')
         self.info = CompositionInfo(source_models)
+        #self.update_trt_model_ids()
```
Comment (ptormene) [inline openquake/commonlib/source.py]: Please remove the comment

File: openquake/commonlib/source.py (hunk 1 above)
Comment (micheles) [inline openquake/commonlib/source.py]: I have added two important comments about why it is necessary to append one source and increment num_ruptures

File: openquake/commonlib/source.py (hunk 2 above)
Comment (micheles) [inline openquake/commonlib/source.py]: I have removed it

**Comment by ptormene** [comment]
LGTM

**Pull request merged**